min-update-versions.json

```diff
@@ -1,5 +1,5 @@
 {
-    "android": "2.11.0",
-    "ios": "2.11.0",
+    "android": "2.33.0",
+    "ios": "2.33.0",
     "web": "0.11.0"
```

Comment:
What about web?

Reply:
iirc On web this actually was never functional 🤔